src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -37,15 +38,15 @@ public function __construct($name = 'debug:twig')
     /**
      * Sets the twig environment.
      *
-     * @param \Twig_Environment $twig
+     * @param Environment $twig
      */
-    public function setTwigEnvironment(\Twig_Environment $twig)
+    public function setTwigEnvironment(Environment $twig)
```

Comment:
We cannot use `Twig` only as all classes need to be under a namespace. That's PSR-4.